juspay/hyperswitch
# feat(user): implement entity level authorization

## Type of Change



## Description
- Created a profile level user.
- Added entity level authorization in routes.



### Additional Changes





## Motivation and Context
Dashboards users to be restricted at profile level.



## How did you test it?
Dashboard UI.

1. Login to dashboard.
2. Invite a user at profile level.
3. Login using profile level user.
4. Check if APIs are working.



## Checklist
